mod_app/models/film_model.py

```diff
@@ -147,5 +149,16 @@ def posters(self):
     def drawings(self):
         return Drawing.objects.filter(film=self)
 
-    comments = models.TextField(blank=True)
-    temporary_images = models.TextField(blank=True)
+    comments = RichTextUploadingField(blank=True)
+    temporary_images = RichTextUploadingField(blank=True)
+
+    bibliography = models.ManyToManyField(BibliographyItem, related_name="films")
+
+    def clean(self, *args, **kwargs):
+        super().clean(*args, **kwargs)
+        validate_format_other(self.format_other, self.format_type)
+
+    def save(self, *args, **kwargs):
+        super().save(*args, **kwargs)
+
+        update_bibliography(self, self.print_comments)
```

Comment:
I will try to remove it from them but otherwise I'd prefer to have confirmation from the researchers if it looks like it's going to result in a lot of new code